README.md

```diff
@@ -1,24 +1,249 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
```

Comment:
### I'm just using the readme file to write comments because not all files are included in this PR you should modify the code that is responsible for what is shown below 

- One of the requirements is to follow the given design by implementing the same colors, font-family, spaces, and layout you can see the screenshot of the requirements below 
<img width="546" alt="Screenshot 2023-02-15 at 15 24 42" src="https://user-images.githubusercontent.com/61790587/219056526-1c6f88ad-759e-45a9-805d-f8aa0f2e4ded.png">

- [ ] it would be better to stick to the design make the sign up page the same as in the design by adding the same font size, and spaces, and layout 

- Sign up page design 
<img width="231" alt="Screenshot 2023-02-15 at 15 27 24" src="https://user-images.githubusercontent.com/61790587/219058245-94929b52-4155-4744-929b-cc779e6bb960.png">

- Yours 
![Screenshot 2023-02-15 at 15 28 06](https://user-images.githubusercontent.com/61790587/219058290-f294a6df-c6e7-441d-984b-bf236649e1ba.png)

- Font family from the Design 
<img width="412" alt="Screenshot 2023-02-15 at 15 25 53" src="https://user-images.githubusercontent.com/61790587/219057802-99df8d3a-3e32-4603-badf-f2e71e8bd4c0.png">

Reply:
- [ ] @NatanimA great work implementing the changes requested however the design needs some changes to match the design given, the input area length and also the button placement below is a comparison,


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/219133672-fc52335b-e052-4890-ac0b-81b15197acda.png)  | ![image](https://user-images.githubusercontent.com/61408860/219133751-953cc7e6-65ed-4a10-a3e8-65da59ff8b8e.png)